compiler/circle-mpqsolver/src/utils/ModuleCloner.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __MPQSOLVER_MODULE_CLONER_H__
+#define __MPQSOLVER_MODULE_CLONER_H__
+
+#include <luci/IR/Module.h>
+
+namespace mpqsolver
+{
+
+class ModuleCloner final
+{
+public:
+  static std::unique_ptr<luci::Module> Clone(const luci::Module *module);
```

Comment:
Method names start with lowercase.
```suggestion
  static std::unique_ptr<luci::Module> clone(const luci::Module *module);
```